Packs/Druva/Integrations/Druva/Druva.py

```
@@ -0,0 +1,358 @@
+    response = clientObj.post_restoreToEndpoint(source_resourceid, target_resourceid, restore_location)
+    statusCode = response.status_code
+    responseJson = response.json()
+    if (statusCode == 200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Restore Job Initiated', responseJson['restores'])
+        outputs = {"Druva.restoreJobs(val.restoreID == obj.restoreID)": responseJson['restores']}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_Restore_Status(clientObj, restore_id):
+    response = clientObj.get_restoreStatus(restore_id)
+    statusCode = response.status_code
+    responseJson = response.json()
+    if (statusCode == 200):
+        responseJson = response.json()
+        readable_output = tableToMarkdown('Restore Job Status', responseJson)
+        outputs = {"Druva.restoreJobs(val.restoreID == obj.restoreID)": responseJson}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
+
+
+def Druva_Decommission(clientObj, resource_id):
+    response = clientObj.post_decommission(resource_id)
+    statusCode = response.status_code
+    responseJson = response.json()
+    if (statusCode == 200):
+        responseJson = response.json()
+        headers = ['Resource ID']
+        readable_output = tableToMarkdown('Device Decomission Request', str(resource_id), headers=headers)
+        outputs = {"Druva.decomissionedResource(val.resource_id == obj.resource_id)": str(resource_id)}
+        raw_response = responseJson
+        return (readable_output, outputs, raw_response)
+    else:
+        demisto.log('Error: ' + str(response.status_code))
+        return None
```

better to raise a proper exception with more information on the error (same for all)

---

Requested Changes made.